cloudigrade/internal/prometheus.py

```diff
@@ -83,6 +88,57 @@ def _initialize_celery_queue_length_metrics(self):
                     partial(redis.execute_command, "llen", [queue_name])
                 )
 
+    def _initialize_db_table_size_metrics(self):
+        """Initialize Database table size metrics."""
+        if not self.db_is_pg():
+            # Currently only supports the Postgres database.
+            return
```

Comment:
I think we should probably put a `logger.warning` before returning here just to be safe.